apps/notifications/emails.py

```diff
@@ -189,7 +189,7 @@ def send_no_object(cls, object, *args, **kwargs):
             ),
             "organisation_id": organisation.id,
         }
-        tasks.send_async_no_object(
+        tasks.send_async_no_object.delay(
             cls.__module__, cls.__name__, object_dict, args, kwargs
```

Comment:
related to a4 celery [PR](https://github.com/liqd/adhocracy4/pull/1493), `object_dict` is not necessary here, so I suggest to update the kwargs instead of defining a new dict `object_dict`.
And I would definitely rename the task to get rid of `...no_object` in it.

Reply:
as discussed today we will postpone any real refactoring to another PR and focus on making celery work. I have there simply removed the return statement.